pkg/fieldpath/fieldpath.go

```diff
@@ -22,6 +22,15 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/meta"
 )
 
+// formatMap formats map[string]string to a string.
+func formatMap(m map[string]string) string {
+	var l string
+	for key, value := range m {
+		l += key + "=" + value + "\n"
```

Comment:
@bgrant0607 to be honest I tried to find out the thread discussion but I couldn't. So I've to admit that I don't remember how I took the assumption that a key=value format could be reasonable. My fault.
I'm OK to modify it in json or yaml format. Sorry about that